ProductsWPF.Integration.Tests/IntegrationTests.cs

```diff
@@ -0,0 +1,43 @@
+using AccessServices.DTOs;
+using AccessServices.Interfaces;
+using DataAccess;
+using Ninject;
+using Ninject.Modules;
+using ProductsWPF.Integration.Tests.Ninject;
+using ProductsWPF.IoC;
+using System;
+using System.Linq;
+using Xunit;
+
+namespace ProductsWPF.Integration.Tests
+{
+    public class IntegrationTests
+    {
+        StandardKernel kernel;
+        public IntegrationTests()
+        {
+            var module = new Bindings();
+            kernel = new StandardKernel(module);
+
+        }
+        [Fact]
+        public void Adding()
+        {
```

Comment:
Try to separate logic within a test by 3 blocks. 
Arrange Act Assert. You can add a single line delimiter or add comments.